### PR TITLE
fix: so swagger json validates

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce?

Fix so swagger json is valid.

### Other information (Screenshots)
Screenshot 2023-08-14 at 11 21 32

### CODE REVIEW

File: apps/api/src/app/events/dtos/trigger-event-request.dto.ts
Comment (djabarovgeorge) [inline apps/api/src/app/events/dtos/trigger-event-request.dto.ts:1]: s this part is fixing the error we had?

Comment (davidsoderberg) [reply apps/api/src/app/events/dtos/trigger-event-request.dto.ts:1]: Yes

File: apps/api/src/app/events/dtos/trigger-event-request.dto.ts [hunk 1]
```diff
@@ -77,6 +69,7 @@ export class TriggerEventRequestDto {
         $ref: getSchemaPath(TopicPayloadDto),
       },
     ],
+    // type: [String, SubscriberPayloadDto, TopicPayloadDto],
```
Comment (p-fernandez) [inline apps/api/src/app/events/dtos/trigger-event-request.dto.ts:72]: Is this meant to be commented out or not?

Comment (davidsoderberg) [reply apps/api/src/app/events/dtos/trigger-event-request.dto.ts:72]: Yes to test CI for now, will put it back soon.

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**